coralproject/talk
# Show overflow text on download comments button when disabled

## What does this PR do?
When comment history has been recently requested and the button is therefore deleted, button text is cutoff on smaller devices. This updates the CSS to display the overflow text.  

## How do I test this PR?
- Request comment history
- Visit Settings panel on smaller device and see Download button now displays all text
